docusaurus.config.js

```diff
@@ -12,6 +12,10 @@ module.exports = {
   organizationName: 'stryker-mutator',
   projectName: 'stryker-mutator.github.io',
   themeConfig: {
+    image: 'images/strykerman.png',
```

Comment:
Does this only have an effect on the meta tags?

Reply:
Yes

Reply:
See https://v2.docusaurus.io/docs/theme-classic/#meta-image